processing/src/test/java/org/apache/druid/query/aggregation/SingleValueAggregationTest.java

```diff
@@ -0,0 +1,254 @@
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation;
+
+import org.apache.druid.common.config.NullHandling;
+import org.apache.druid.error.DruidException;
+import org.apache.druid.segment.ColumnSelectorFactory;
+import org.apache.druid.segment.TestHelper;
+import org.apache.druid.segment.column.ColumnCapabilities;
+import org.apache.druid.segment.column.ColumnType;
+import org.apache.druid.segment.column.ValueType;
+import org.easymock.EasyMock;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+/**
+  */
+public class SingleValueAggregationTest
+{
+  private SingleValueAggregatorFactory longAggFatory;
+  private ColumnSelectorFactory colSelectorFactoryLong;
+  private ColumnCapabilities columnCapabilitiesLong;
+  private TestLongColumnSelector selectorLong;
+
+  private SingleValueAggregatorFactory doubleAggFatory;
+  private ColumnSelectorFactory colSelectorFactoryDouble;
+  private ColumnCapabilities columnCapabilitiesDouble;
+  private TestDoubleColumnSelectorImpl selectorDouble;
+
+  private SingleValueAggregatorFactory stringAggFatory;
+  private ColumnSelectorFactory colSelectorFactoryString;
+  private ColumnCapabilities columnCapabilitiesString;
+  private TestObjectColumnSelector selectorString;
+
+  private final long[] longValues = {9223372036854775802L, 9223372036854775803L};
+  private final double[] doubleValues = {5.2d, 2.8976552d};
+  private final String[] strValues = {"str1", "str2"};
+
+  public SingleValueAggregationTest() throws Exception
+  {
+    String longAggSpecJson = "{\"type\": \"singleValue\", \"name\": \"lng\", \"fieldName\": \"lngFld\", \"columnType\": \"LONG\"}";
+    longAggFatory = TestHelper.makeJsonMapper().readValue(longAggSpecJson, SingleValueAggregatorFactory.class);
+
+    String doubleAggSpecJson = "{\"type\": \"singleValue\", \"name\": \"dbl\", \"fieldName\": \"dblFld\", \"columnType\": \"DOUBLE\"}";
+    doubleAggFatory = TestHelper.makeJsonMapper().readValue(doubleAggSpecJson, SingleValueAggregatorFactory.class);
+
+    String strAggSpecJson = "{\"type\": \"singleValue\", \"name\": \"str\", \"fieldName\": \"strFld\", \"columnType\": \"STRING\"}";
+    stringAggFatory = TestHelper.makeJsonMapper().readValue(strAggSpecJson, SingleValueAggregatorFactory.class);
+  }
+
+  @Before
+  public void setup()
+  {
+    NullHandling.initializeForTests();
```

Comment:
nit: Instead of calling it in the setup(), we can subclass the test class with `extends InitializedNullHandlingTest`